nifi-nar-bundles/nifi-evtx-bundle/nifi-evtx-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,35 @@
+nifi-evtx-nar
+Copyright 2016 The Apache Software Foundation
+
+This product includes the following work from python-evtx (https://github.com/williballenthin/python-evtx):
```

Comment:
lines 4-8 are not needed here.  This is necessary for the NOTICE of the source release which is the top level nifi/NOTICE file but this NOTICE file within the nar is only for the binary distribution of the nar and that does not contain these sources.